ee/server/service/devices.go

```diff
@@ -110,6 +110,7 @@ func (svc *Service) GetFleetDesktopSummary(ctx context.Context) (fleet.DesktopSu
 	// organization information
 	sum.Config.OrgInfo.OrgName = appCfg.OrgInfo.OrgName
 	sum.Config.OrgInfo.OrgLogoURL = appCfg.OrgInfo.OrgLogoURL
+	sum.Config.OrgInfo.OrgLogoURLLightBackground = appCfg.OrgInfo.OrgLogoURLLightBackground
```

Comment:
could you please also add this to the corresponding integration tests?

We have some tests for the device endpoint at `server/service/integration_desktop_test.go` and tests specific to the migration in `server/service/integration_mdm_test.go`.

Ideally at least one of the tests will involve:

1. Getting the config with the default value if is not set
2. Doing a `PATCH` request to set the config
3. Getting the config with the modified value